releasenotes.md

```diff
@@ -0,0 +1,7 @@
+# 3.0.0.2
```

Comment:
>3.0.0.2 [](start = 2, length = 7)

Does 3.0.0.2 work with older version of OneGet? If not, we need to let user know what to do.